apidef/oas/middleware.go

```diff
@@ -71,15 +75,37 @@ func (g *Global) ExtractTo(api *apidef.APIDefinition) {
 }
 
 type CORS struct {
-	Enabled            bool     `bson:"enabled" json:"enabled"` // required
-	MaxAge             int      `bson:"maxAge,omitempty" json:"maxAge,omitempty"`
-	AllowCredentials   bool     `bson:"allowCredentials,omitempty" json:"allowCredentials,omitempty"`
-	ExposedHeaders     []string `bson:"exposedHeaders,omitempty" json:"exposedHeaders,omitempty"`
-	AllowedHeaders     []string `bson:"allowedHeaders,omitempty" json:"allowedHeaders,omitempty"`
-	OptionsPassthrough bool     `bson:"optionsPassthrough,omitempty" json:"optionsPassthrough,omitempty"`
-	Debug              bool     `bson:"debug,omitempty" json:"debug,omitempty"`
-	AllowedOrigins     []string `bson:"allowedOrigins,omitempty" json:"allowedOrigins,omitempty"`
-	AllowedMethods     []string `bson:"allowedMethods,omitempty" json:"allowedMethods,omitempty"`
+	// If set to `true`, this option turns CORS processing on or off.
+	// Old API definition: `CORS.enable`.
+	Enabled bool `bson:"enabled" json:"enabled"` // required
```

Comment:
It'll be better to have the godoc pattern used here right? Comment starting with field name

Reply:
For sure. Added godoc comments for complete oas package in #4327, moved everything there.